TopMovies/Core/State/UpdateRequestState.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  UpdateRequestState.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 24.03.2021.
+//
+
+enum UpdateRequestState: AutoEnum {
+  case requested, downloading, updated
+  case failed(Error)
+}
```

Comment:
Do we even need that?
I feel that `updated` is not used anywhere, so you could use some base request state